pkg/cli-runtime/config_test.go

```diff
@@ -153,3 +153,47 @@ func TestConfig_Print(t *testing.T) {
 		})
 	}
 }
+func TestConfig_EmojiPrint(t *testing.T) {
```

Comment:
You can merge these 2 tests into 1. You can set the printer `EmojiBoldf` or `EmojiSuccessf` in the test case. You can avoid the duplication

Reply:
Done